ta/optee/sedget_video_ta.c

```diff
@@ -164,7 +164,7 @@ static uint8_t *get_phys_address(TEE_Param *param)
 	}
 
 	/* LIMIT: current mve mmu cares for 32bit address */
-	return (uint8_t *)(uintptr_t)p[1].value.a;
+	return (uint8_t *)((uintptr_t)p[1].value.a << 32 | p[1].value.b);
```

Comment:
I guess the `(uintptr_t)` cast is just for `p[1].value.a`? `p[1].value.b` is just a plain value here?

Reply:
Actually, p[1].value.a is empty; p[1].value.b is an value.

In draft commit we treat a as valid value; and b is empty; and they got reversed in final commit.
https://github.com/OP-TEE/optee_os/commit/7f45f761df896de9d0e11788225fd87a1da1a170

And we need to treat whole values as 64bit even current only b has values; so a is involved even it is empty for current.